### DISCUSSION TITLE
Why the package size is 5kb?

### DISCUSSION THREAD

**Discussion by xxxsf** [preactjs/preact#4062] [Q&A]
![image](https://github.com/preactjs/preact/assets/14307551/c946e94b-3507-4d34-acc2-162a4307a883)

May I ask how is the package volume calculated? Just curious

**Comment by marvinhagemeister** [comment]
Not sure what `jsdelivr` is doing. The size is picked from our build tools where we bundle + minify the files and calculate the compressed size locally. Currently it looks like this for me:
![Screenshot 2023-07-07 at 15 06 15](https://github.com/preactjs/preact/assets/1062408/f8f095ce-1bc5-40d2-9f02-e115148736b4)

**Reply by rschristian** [reply]
Just to add: size in the network pane of all browsers' devtools is the combination of response headers and response data. So that `5.0kb` is the total size of everything sent back to the browser and therefore not a very useful number. The very same asset could have very different sizes associated with it depending on the server that's hosting it.

**Comment by xxxsf** [comment]
Got it，Thx Bro. @marvinhagemeister @rschristian
